### DISCUSSION TITLE
why window.history.state.idx changes to NaN?

### DISCUSSION THREAD

**Discussion by thkim9012** [remix-run/react-router#9067] [Q&A]
when navigate path with url has ended  pound sign, window.history.state.idx changes to NaN 

I don't know why it is NaN..

**Comment by takapiro99** [comment]
experiencing the same problem, it occurs very rarely and randomly. (can't reproduce)
can't figure out why it happens and even if caused by react-router.
I found it happens on mac and windows chrome (with very random page transitions)

> when navigate path with url has ended pound sign

the problem happens even if not ends with pound sign in my environment

**Comment by tgyou** [comment]
@thkim9012 
downgrade to 6.3.0.
I have also been having problems since the 6.4.0 update.

**Reply by thkim9012** [reply]
@tgyou It is the same in v6.3.0.

**Comment by thkim9012** [comment]
it causes trouble for "useBlocker" and "usePrompt" which is removed in v6.
